internal/metamorph/processor.go

```diff
@@ -754,6 +734,7 @@ func (p *Processor) ProcessTransaction(ctx context.Context, req *ProcessorReques
 	tracing.EndTracing(responseProcessorAddSpan, nil)
 }
 
+// process txs submitted to message queue
```

Comment:
Change this comment to

`ProcessTransactions processes txs submitted to message queue`